pom.xml

```diff
@@ -1826,6 +1830,26 @@
       </properties>
     </profile>
 
+    <profile>
+      <!--
+          Use this profile only for making Spark releases. Note that due to SPARK-8819,
+          you must use maven version 3.2.x or before to avoid running into MSHADE-148.
+      -->
+      <id>release</id>
```

Comment:
Maven's release plugin already uses a profile called `release-profile` for stuff like this. Can we use the same name? it's automatically enabled when releasing.
http://maven.apache.org/maven-release/maven-release-plugin/perform-mojo.html